gtsfm/utils/geometry_comparisons.py

```diff
@@ -369,3 +369,17 @@ def get_points_within_radius_of_cameras(
     is_nearby_to_any_cam = np.any(is_nearby_matrix, axis=1)
     nearby_points_3d = points_3d[is_nearby_to_any_cam]
     return nearby_points_3d
+
+
+def angle_between_vectors(v_a: np.ndarray, v_b: np.ndarray) -> float:
```

Comment:
we already have this function in one of the utils file. Can you please check there.

Reply:
it looks like `compute_relative_unit_translation_angle(U_1: Optional[Unit3], U_2: Optional[Unit3]) -> Optional[float]:` will do the trick, if we just wrap the arguments with
```python
angle_between_vectors(Unit3(v_a), Unit3(v_b))
```